_sass/_components/post-details.scss

```diff
@@ -37,20 +45,25 @@
 
 .post-tags {
   display: block;
-  margin-bottom: 3rem !important;
+  margin-top: units(2);
+
   a {
-    @include u-padding('05');
-    @include u-margin('05');
+    padding: units(2);
```

Comment:
could we make this units(1) instead of 2

Reply:
👍